generators/client/templates/angular/src/test/javascript/spec/app/account/settings/settings.component.spec.ts.ejs

```diff
@@ -58,7 +47,7 @@ describe('Component Tests', () => {
         beforeEach(() => {
             fixture = TestBed.createComponent(SettingsComponent);
             comp = fixture.componentInstance;
-            mockAuth = fixture.debugElement.injector.get(AccountService);
+            mockAuth = (fixture.debugElement.injector.get(AccountService) as unknown) as MockAccountService;
```

Comment:
Can't we use TestBed.get(AccountService) ?

Reply:
Great hint, thanks! I made change and force pushed.